classes/ETL/Ingestor/HostTableTransformIngestor.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace ETL\Ingestor;
+
+use ETL\aOptions;
+use ETL\iAction;
+use ETL\aAction;
+use ETL\Configuration\EtlConfiguration;
+use ETL\EtlOverseerOptions;
+
+use Log;
+
+class HostTableTransformIngestor extends pdoIngestor implements iAction
+{
+
+    private $_instance_state;
```

Comment:
Each variable needs a docblock describing what it does and the expected values. For example:
```
/**
 * @var string Regex used to identify variables in the source string.
 */
private $variableRegex = '/\${([^}]+)}/';
```